git-worktree/src/lib.rs

```diff
@@ -1 +1,126 @@
 #![forbid(unsafe_code, rust_2018_idioms)]
+//! Git Worktree
+
+use git_hash::oid;
+use git_object::bstr::ByteSlice;
+use quick_error::quick_error;
+use std::convert::TryFrom;
+use std::fs;
+use std::fs::create_dir_all;
+use std::path::Path;
+use std::time::Duration;
+
+#[cfg(unix)]
+use std::os::unix::fs::PermissionsExt;
+
+quick_error! {
+    #[derive(Debug)]
+    pub enum Error {
+        Utf8Error(err: git_object::bstr::Utf8Error) {
+            from()
+            display("Could not convert path to UTF8: {}", err)
+        }
+        TimeError(err: std::time::SystemTimeError) {
+            from()
+            display("Could not read file time in proper format: {}", err)
+        }
+        ToU32Error(err: std::num::TryFromIntError) {
+            from()
+            display("Could not convert seconds to u32: {}", err)
+        }
+        IoError(err: std::io::Error) {
+            from()
+            display("IO error while writing blob or reading file metadata or changing filetype: {}", err)
+        }
+        FindOidError(oid: git_hash::ObjectId, path: std::path::PathBuf) {
```

Comment:
For consistency, this should be called `NotFound { oid: git_hash::ObjectId, path: PathBuf}`.

Reply:
Fixed: https://github.com/Byron/gitoxide/pull/315/commits/eaee85595dc658549e62e3292b025ec016e70abd